http/src/main/resources/jupyter/gce-init.sh

```diff
@@ -0,0 +1,528 @@
+#!/usr/bin/env bash
```

Comment:
can we move this file outside `jupyter` dir?

Reply:
Why? Our current pattern is to use the `jupyter` dir for resources that need to be uploaded to the init bucket.

Reply:
My intuition would be to expect only jupyter-specific files inside the jupyter directory, no? There are other files that'd need to be moved out with that logic, though so perhaps it'd be better to do the reorganization for all later?

Reply:
if these are jupyter specific resources, it makes sense to be under `jupyter`. If not, shouldn't we move away from having these files in jupyter?...`jupyter` looks pretty overloaded.

Reply:
If the common denominator for all is uploading to the init bucket, we could also rename the directory as `init-bucket` or something?

Reply:
The `jupyter` dir today is for all resources that are updated to the init bucket. We could rename it; also need to change this line: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/ClusterResourcesConfig.scala#L18